src/main/java/bio/terra/app/configuration/WebConfig.java

```diff
@@ -34,8 +38,23 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 
   @Override
   public void addResourceHandlers(ResourceHandlerRegistry registry) {
-    registry
-        .addResourceHandler("/webjars/swagger-ui-dist/**")
-        .addResourceLocations("classpath:/META-INF/resources/webjars/swagger-ui-dist/4.3.0/");
+    Properties properties = new Properties();
+    try (InputStream propsFile =
+        getClass().getClassLoader().getResourceAsStream("swagger-ui.properties")) {
+      properties.load(propsFile);
+      String swaggerUIVersion = properties.getProperty("swagger-ui");
+      if (swaggerUIVersion == null) {
+        throw new RuntimeException(
+            "Cannot find Swagger UI version from swagger-ui.properties, please make sure it is configured correctly in build.gradle.");
+      }
```

Comment:
Instead of this, I would use `requireNonNull()` and add `NullPointerException` to the `catch` below.
```suggestion
      String swaggerUIVersion = Objects.requireNonNull(properties.getProperty("swagger-ui"));
```